app/src/main/java/org/sopt/and/data/dataremote/datasourceimpl/HobbyRemoteDataSourceImpl.kt

```diff
@@ -0,0 +1,13 @@
+package org.sopt.and.data.dataremote.datasourceimpl
+
+import org.sopt.and.data.dataremote.datasource.HobbyRemoteDataSource
+import org.sopt.and.data.dataremote.model.response.BaseResponse
+import org.sopt.and.data.dataremote.model.response.ResponseHobbyDto
+import org.sopt.and.data.dataremote.service.UserService
+import javax.inject.Inject
+
+class HobbyRemoteDataSourceImpl @Inject constructor(
+    private val userService: UserService
+):HobbyRemoteDataSource{
```

Comment:
kotlin 컨벤션에 따르면 이렇게 수정이 될거 같은데, 어떤 이유로 이렇게 수정해야하는지에 대해서 한번 알아보시는것도 좋을거 같습니다!!
```suggestion
) : HobbyRemoteDataSource{
```

Reply:
Option + Command + L !!!!